test/cmd/newapp.sh

```diff
@@ -612,5 +610,23 @@ os::cmd::expect_success 'oc new-app php-55-centos7:latest https://github.com/ope
 os::cmd::expect_success_and_text 'oc get bc test8 --template={{.spec.strategy.sourceStrategy.from.name}}' 'php-55-centos7:latest'
 os::cmd::expect_success 'oc delete project new-app-syntax'
 
+# new-app docker build strategy with binary input
+os::cmd::expect_success 'oc project ${default_project}'
+os::cmd::expect_success 'oc delete all,templates --all'
+os::cmd::expect_success 'oc create -f examples/image-streams/image-streams-centos7.json'
+os::cmd::try_until_success 'oc get imagestreamtags ruby:2.0' # need to wait until tags are available!?
```

Comment:
you should at least be waiting until the tag you care about is available.  2.0 is a bad choice given how old it is.

Reply:
it's intended, polling/waiting until the tag is imported is the right thing to do.

Reply:
ok